backend/local.env

```diff
@@ -19,3 +19,7 @@ JWT_REFRESH_TOKEN_EXPIRATION_TIME=31540000
 # JWT_REFRESH_TOKEN_EXPIRATION_TIME=10
 
 ACTIVE_PROFILES=test_data,e2e
+
+FRONTEND_URL=http://localhost:3000
+EMAIL_USERNAME=teampnrn@gmail.com
+EMAIL_PASSWORD=soppysknponsqjow
```

Comment:
ideally, we would create test emails to make this more secure but didn't want to create a new email, maybe in the future 🤔